### DISCUSSION TITLE
Settings not saving 1.6.3

### DISCUSSION THREAD

**Discussion by FPVSO** [OrcaSlicer/OrcaSlicer#1395] [General]
Hello,

I run as admin and still when I close out and rerun program - all my settings / progress will not reload even if I try to save it.  How do I correct this problem?

**Comment by ghost** [comment]
I have the same issue :/

**Reply by fr3ak2402** [reply]
Am already in the process of fixing that

**Reply by fr3ak2402** [reply]
Are you using the profiles that are shipped by default with version V 1.6.3?

**Reply by ghost** [reply]
Yes i am.

**Reply by fr3ak2402** [reply]
Okay... thanks for the feedback.

Did you switch from version 1.6.2 to version V 1.6.3?

**Reply by ghost** [reply]
Yes i'm 99,9% certain i did

**Reply by fr3ak2402** [reply]
Then I will just reproduce the problem.

Which printers do you use in OrcaSlicer?

**Reply by ghost** [reply]
Cr10 V2 and Mercury Zero G

**Reply by fr3ak2402** [reply]
Thank you, for the feedback.

I was able to fix the problem in the meantime... 
I'm already finishing a patch...

**Reply by FPVSO** [reply]
Hello,

Thanks for the replies.  This is a new-to-me app (thanks for doing this btw I like it a lot other than my saving issue).

I use Ender 3 S1 (not pro)

**Reply by ghost** [reply]
yo that was quick! thx dude, love this slicer. Keep it up :D

**Reply by enewbold** [reply]
Same for me with the Flsun V400.  Most of the settings are saved, but not all of them.  I like my first layer height to be 0.3, but every time I open Orca Slicer it has gone back to 0.2.  Thank you.

**Reply by FPVSO** [reply]
Still having issue with Ender 3 S1

**Comment by FPVSO** [comment]
> Are you using the profiles that are shipped by default with version V 1.6.3?

Ender 3 S1

**Comment by FPVSO** [comment]
Still having same issues (Ender 3 S1)

**Comment by FPVSO** [comment]
Okay.  So I'll follow up on my own thread since this is incredibly lacking-of other user(s) feedback.

I ended up reformatting my computer and now it saves no problems.  I'm not as 'code-savvy, so I don't know exactly what happened.  Likely it's something to do with the method of saving was not working correctly and/or the directory of how it saved was messed up.

Maybe this little detail will help OTHERS who may be in the same situation as me.  I tested this theory on putting this slicer on a laptop that has never had OrcaSlicer installed.  

This would suggest (maybe) that if you had any previous versions of OrcaSlicer this could happen.

If anyone can shine a light on this- this would be fantastic.

**Reply by ghost** [reply]
Yeah i got it to work after i cleard my AppData maps of all traces of my older versions of Orca, Something in there is screwing with the newest version.

Hope this helps other users with the same problem
